docs/quickstart.rst

```diff
@@ -0,0 +1,385 @@
+
+Now, your Token is fully set up and will be used in any API requests made during the execution of this script.
+
+If you want to use the same Token in future scripts, you can save it to the Config file:
+
+.. code-block:: python
+
+    token.save_token()
+
+Now, the token will be automatically loaded every time you import pyrefinebio.
+
+If you want to get access to the token that is saved to the config file in the future, you can load it:
+
+.. code-block:: python
+
+    token = pyrefinebio.Token.load_token()
+
+For more information check out the :ref:`Token` class documentation and :ref:`Config`.
+
+Downloading Datasets
+--------------------
+
+After you set up and activate a Token you can start creating and downloading Datasets.
+
+pyrefinebio provides the function `download_dataset()` that makes creating and downloading Datasets super easy.
+It will will automatically handle every part of the creation and download process for you.
+
+`download_dataset()` requires that you pass in the parameters `path`, `email_address`, and either `dataset_dict` or `experiments`.
+
+* `path` is the path that the Dataset will be downloaded to. This can be a path to a directory, in which case a filename will automatically be created, or a path to a zip file. 
+
+* `email_address` is the email address that will be notified when the Dataset is finished processing.
+
+The parameters `dataset_dict` and `experiments` both control the Experiments and Samples that will be a part of the Dataset.
+
+* `dataset_dict` should be used when you want to specify specific Samples to be included in the Dataset. It should be in the format:
+
+.. code-block:: python
+
+    dd = {
+        "Experiment 1 Accession Code": [
+            "Sample 1 Accession Code",
+            "Sample 2 Accession Code"
+        ],
+        "Experiment 2 Accession Code": [
+            "Sample 3 Accession Code",
+            "Sample 4 Accession Code"
+        ]
+    }
+
+* `experiments` can be used when you only care about experiments. All the samples associated with the Experiments that you pass in that are downloadable will be added to the Dataset. It is just a list of Experiment accession codes or pyrefinebio Experiment objects:
+
+.. code-block:: python
+
+    ex2 = pyrefinebio.Experiment.get("Experiment 2 Accession Code")
+
+    exs = ["Experiment 1 Accession Code", ex2]
+
+You can also pass in other optional parameters to alter the Dataset itself and to alter how the download process works.
+
+* `aggregation` can be used to change how the Dataset is aggregated.
+
+* `transformation` can be used to change the transformation of the Dataset.
+
+* `skip_quantile_normalization` can be used to choose whether or not quantile normalization is skipped.
+
+* `extract` can be used to choose whether the downloaded zip file should be automatically extracted.
+
+* `prompt` can be used to choose whether or not you should be prompted before downloading if the Dataset zip file is larger than 1 gigabyte.
```

Comment:
We should list the options/defaults for these parameters. We should have the string literals they can choose from listed (found [here](https://github.com/AlexsLemonade/refinebio/blob/dev/common/data_refinery_common/models/dataset.py#L22)) along with links to the docs about [aggregations](http://docs.refine.bio/en/latest/main_text.html?highlight=aggregation#aggregations), [skipping normalization](http://docs.refine.bio/en/latest/main_text.html?highlight=aggregation#skipping-quantile-normalization-for-rna-seq-experiments), and [transformations](http://docs.refine.bio/en/latest/main_text.html?highlight=aggregation#gene-transformations).

We should also answer the question:
> If I set `extract=True`, where will it be extracted to?

Reply:
added extra info